runtime/runtime/src/verifier.rs

```diff
@@ -2295,4 +2332,50 @@ mod tests {
         // cspell:ignore привет
         check("привет", 3, "п");
     }
+
+    #[test]
```

Comment:
optional: Could you please add a test for valid action and for invalid action to fail somewhere in validate_access_key_permission - to make sure that those checks keep happening as well?